drivers/flash/spi_nor.c

```diff
@@ -1143,6 +1147,23 @@ static int spi_nor_configure(const struct device *dev)
 		return -ENODEV;
 	}
 
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
+	if (cfg->reset) {
+		if (!device_is_ready(cfg->reset->port)) {
+			LOG_ERR("Reset pin not ready");
+			return -ENODEV;
+		}
+		if (gpio_pin_configure_dt(cfg->reset, GPIO_OUTPUT_ACTIVE)) {
+			LOG_ERR("Couldn't configure reset pin");
+			return -ENODEV;
+		}
+		rc = gpio_pin_set_dt(cfg->reset, 0);
+		if (rc) {
+			return rc;
+		}
```

Comment:
I guess what happens here is the reset i set active and then inactive immediately after. Shouldn't there be some configurable delay between these two?

Reply:
On the flash model we are using reset time is 1uS.
But yes it could be configurable in device tree.